user_docs/en/userGuide.t2t

```diff
@@ -390,8 +391,7 @@ When moving the review cursor, the System caret does not follow along, so you ca
 However, by default, when the System caret moves, the review cursor follows along.
 This can be toggled on and off.
 
-Note that braille follows the [focus #SystemFocus] and [caret #SystemCaret] by default, rather than object navigation and text review.
-If you want it to follow object navigation and text review instead, you need to [configure braille to be tethered to #BrailleTether] review.
+Note that by default, braille tries to follow automatically both the system carret and the review cursor. This can be configured in the [Braille Tether section #BrailleTether].

```

Comment:
Similar problems here. Again I would just point to the braille tether section of the user guide.

```suggestion
Note: Braille following the review cursor can be configured via [Braille Tether #BrailleTether].
```